manifests/kustomize/env/aws/params.env

```diff
@@ -0,0 +1,6 @@
+dbHost=YOUR_RDS_ENDPOINT
+mlmdDb=rds
```

Comment:
any reason to change `mlmdDb` here?

Reply:
Not really, it's just a name which indicate your Database name.